### PR TITLE
Update translation for docs.gaphor.org

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Gaphor/docs.gaphor.org](https://hosted.weblate.org/projects/gaphor/docs-gaphor-org/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 1]
```diff
@@ -7317,58 +7317,64 @@ msgid ""
 "diagram. When you set a `color`` or a `font-family` on `diagram`, it will "
 "propagate down to the items contained in the diagram."
 msgstr ""
+"Certaines propriétés sont héritées du style parent. Le parent est souvent un "
+"diagramme. Lorsque vous définissez une `couleur` ou une `famille de "
+"caractères` sur `diagramme`, cela se propagera aux éléments contenus dans le "
+"diagramme."
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7323]: `color` and `font-family` are CSS keywords.

```suggestion
"Certaines propriétés sont héritées du style parent. Le parent est souvent un "
"diagramme. Lorsque vous définissez une `color` (couleur) ou une `font-family` "
"(famille de caractères) sur `diagram`, cela se propagera aux éléments contenus dans le "
"diagramme."
```

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 2]
```diff
@@ -7317,58 +7317,64 @@ msgid ""
...
 
 #: ../style_sheets.md:140
 msgid "Colors"
-msgstr ""
+msgstr "Couleurs"
 
 #: ../style_sheets.md:144
 msgid "``background-color``"
-msgstr ""
+msgstr "``couleur d'arrière-plan``"
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7331]: `background-color` is a keyword. It should not be translated.

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 3]
```diff
@@ -7317,58 +7317,64 @@ msgid ""
...
 
 #: ../style_sheets.md:144
 msgid "Examples:"
-msgstr ""
+msgstr "Exemples :"
 
 #: ../style_sheets.md:146
 msgid "``background-color: azure;``"
-msgstr ""
+msgstr "``couleur d'arrière-plan : azur ;``"
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7339]: Same here.

```suggestion
msgstr "``background-color: azure;``"
```

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 4]
```diff
@@ -7317,58 +7317,64 @@ msgid ""
...
 
 #: ../style_sheets.md:148
 msgid "``background-color: rgb(255, 255, 255);``"
-msgstr ""
+msgstr "``couleur d'arrière-plan : rgb(255, 255, 255) ;``"
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7343]: Same here.

```suggestion
msgstr "``background-color: rgb(255, 255, 255);``"
```

etc.

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 5]
```diff
@@ -7317,58 +7317,64 @@ msgid ""
...
 
 #: ../style_sheets.md:150
 msgid "``background-color: hsl(130, 95%, 10%);``"
-msgstr ""
+msgstr "``couleur d'arrière-plan : hsl(130, 95%, 10%) ;``"
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7347]:
```suggestion
msgstr "``background-color: hsl(130, 95%, 10%);``"
```

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 6]
```diff
@@ -7317,58 +7317,64 @@ msgid ""
...
 
 #: ../style_sheets.md:151
 msgid "``color``"
-msgstr ""
+msgstr "``couleur``"
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7351]:
```suggestion
msgstr "``color``"
```

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 7]
```diff
@@ -7317,58 +7317,64 @@ msgid ""
...
 
 #: ../style_sheets.md:151
 msgid "Color used for lines. *(inherited)*"
-msgstr ""
+msgstr "Couleur utilisée pour les lignes. *(hérité)*"
 
 #: ../style_sheets.md:152
 msgid "``text-color``"
-msgstr ""
+msgstr "``couleur-texte``"
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7359]:
```suggestion
msgstr "``text-color``"
```

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 8]
```diff
@@ -7317,58 +7317,64 @@ msgid ""
...
 
 #: ../style_sheets.md:154
 msgid "Use color if possible."
-msgstr ""
+msgstr "Utilisez la couleur si possible."
 
 #: ../style_sheets.md:155
 msgid "``opacity``"
-msgstr ""
+msgstr "``opacité``"
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7371]:
```suggestion
msgstr "``opacity``"
```

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 9]
```diff
@@ -7377,264 +7383,294 @@ msgid ""
...
 
 #: ../style_sheets.md:170
 msgid "``font-style``"
-msgstr ""
+msgstr "``font-style``"
 
 #: ../style_sheets.md:170
 msgid "Either ``normal`` or ``italic``. *(inherited)*"
-msgstr ""
+msgstr "Soit ``normal`` ou ``italique``. *(hérité)*"
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7424]:
```suggestion
msgstr "Soit ``normal`` ou ``italic`` (italique). *(hérité)*"
```

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 10]
```diff
@@ -7377,264 +7383,294 @@ msgid ""
...
 
 #: ../style_sheets.md:171
 msgid "``font-weight``"
-msgstr ""
+msgstr "``font-weight``"
 
 #: ../style_sheets.md:171
 msgid "Either ``normal`` or ``bold``. *(inherited)*"
-msgstr ""
+msgstr "Soit ``normal`` ou ``gras``. *(hérité)*"
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7432]:
```suggestion
msgstr "Soit ``normal`` ou ``bold`` (gras). *(hérité)*"
```

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 11]
```diff
@@ -7377,264 +7383,294 @@ msgid ""
...
 
 #: ../style_sheets.md:172
 msgid "``text-align``"
-msgstr ""
+msgstr "``text-align``"
 
 #: ../style_sheets.md:172
 msgid "Either ``left``, ``center``, ``right``. *(inherited)*"
-msgstr ""
+msgstr "Soit ``gauche``, ``centre``, ``droite``. *(hérité)*"
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7440]:
```suggestion
msgstr "Soit ``left`` (gauche), ``center`` (centre), ``right`` (droite). *(hérité)*"
```

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 12]
```diff
@@ -7377,264 +7383,294 @@ msgid ""
...
 
 #: ../style_sheets.md:173
 msgid "``text-decoration``"
-msgstr ""
+msgstr "``text-decoration``"
 
 #: ../style_sheets.md:173
 msgid "Either ``none`` or ``underline``."
-msgstr ""
+msgstr "Soit ``non`` ou ``souligné``."
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7448]:
```suggestion
msgstr "Soit ``none`` (rien) ou ``underline`` (souligné)."
```

File: docs/locale/fr/LC_MESSAGES/docs.po [hunk 13]
```diff
@@ -7377,264 +7383,294 @@ msgid ""
...
 
 #: ../style_sheets.md:174
 msgid "Vertical alignment for text."
-msgstr ""
+msgstr "Alignement vertical du texte."
 
 #: ../style_sheets.md:176
 msgid "Either ``top``, ``middle`` or ``bottom``."
-msgstr ""
+msgstr "Soit ``haut``, ``milieu`` ou ``bas``."
```
Comment (amolenaar) [inline docs/locale/fr/LC_MESSAGES/docs.po:7460]:
```suggestion
msgstr "Soit ``top`` (haut), ``middle`` (milieu) or ``bottom`` (bas)."
```

**Review by amolenaar — CHANGES_REQUESTED** [review]
In a couple of cases (there's also ones that are okay), the english CSS terms are translated.
If you would apply those to a style sheet, it will not work.

I suggest to use the english terms and add a translation in brackets.

I proposed a couple of changes, but there are more cases than the ones I addressed.

**Comment by amolenaar** [comment]
@stansz84 I think some keywords shouldn't have been translated (e.g. `background-color` from CSS). I added some suggestions in the comments.

**Comment by stansz84** [comment]
@amolenaar I've corrected all the words between the `quotes`.

**Review by amolenaar — APPROVED** [review]

**Comment by amolenaar** [comment]
Thanks @stansz84. This is a lot better.

I also added the changes I already proposed.

**Pull request merged**